lms/djangoapps/discussion/static/discussion/js/views/discussion_board_view.js

```diff
@@ -138,22 +138,23 @@
                 var amount, browseFilterHeight, discussionBottomOffset, discussionsBodyBottom,
                     discussionsBodyTop, headerHeight, refineBarHeight, scrollTop, sidebarHeight, topOffset,
                     windowHeight, $discussionBody, $sidebar;
+                var sidebar_padding = 10;
```

Comment:
If we replace this line in `render` method of `DiscussionBoardView`
```
$(window).bind('load scroll resize', this.updateSidebar);
```
with this
```
$(window).bind('load scroll resize', $.proxy(this.updateSidebar, this));
```
we would be able to pass  proper `this` context to `updateSidebar` method and we would not need to make all those changes.
If this works we need to push this change in upstream too.